samples/chaincode/cc-tools-demo/README.md

```diff
@@ -0,0 +1,332 @@
+# CC-Tools-Demo Tutorial
+
+This tutorial shows how to build, install and test a Go Chaincode developed using the [CC-Tools](https://github.com/hyperledger-labs/cc-tools) framework and integrating it with the Fabric Private Chaincode (FPC) framework.
+
+This tutorial illustrates a simple use case where we follow the [cc-tools-demo](https://github.com/hyperledger-labs/cc-tools-demo) chaincode which is based on standard Fabric and then convert it to an FPC chaincode achieving FPC security capabilities.
+
+This tutorial is based on the [FPC with CC-Tools integration project](https://lf-hyperledger.atlassian.net/wiki/spaces/INTERN/pages/21954957/Hyperledger+Fabric+CC-Tools+Support+for+Fabric+Private+Chaincode) and all our design choices are explained here in the [design document](https://github.com/hyperledger/fabric-private-chaincode/tree/main/docs/design/integrate-with-cc-tools).
+Here are the steps to accomplish this:
+
+* Clone the cc-tools-demo chaincode
+* Modify the chaincode to use FPC
+* Build your FPC CC-tools-demo chaincode
+* Launch a Fabric network
+* Install and instantiate your chaincode
+* Invoke transactions by using the FPC simple-cli
+
+## Prerequisites
+
+* This tutorial presumes that you have installed FPC as described in the FPC [README.md](../../../README.md#clone-fabric-private-chaincode) and `$FPC_PATH` is set accordingly.
+* We need a working FPC development environment. As described in the "Setup your Development Environment" Section of the FPC [README.md](../../../README.md#setup-your-development-environment), you can use our docker-based dev environment (Option 1) or setup your local development environment (Option 2).
+  We recommend using the docker-based development environment and continue this tutorial within the dev container terminal.
+* Moreover, within your FPC development you have already installed the FPC Go Chaincode Support components.
+  See the installation steps in [ecc_go/README.md](../../../ecc_go/README.md#installation).
+* We assume that you are familiar with Fabric chaincode development in go.
+  Most of the steps in this tutorial follow the normal Fabric chaincode development process, however, there are a few differences that we will highlight here.
+* Also, since the tutorial is on the integration between cc-tools and FPC, we expect you to have a grasp knowledge of [cc-tools](https://github.com/hyperledger-labs/cc-tools) framework and that you've at least tried to run the [cc-tools-demo](https://github.com/hyperledger-labs/cc-tools-demo) once by yourself on a Fabric network
+
+## Clone the cc-tools-demo chaincode
+
+We need to clone the chaincode folder from the [cc-tools-demo](https://github.com/hyperledger-labs/cc-tools-demo) repository here.
+
+**Note**: If you're not inside the dev environment, git commands will need sudo permissions like the following:
```

Comment:
I have the feeling that using `sudo` here depends on the particular "developer environment". How someone is using the docker dev environment, etc ... 
Thus, I would not advise the user to run the things as sudo here. Maybe just give the note, that if you experience permission errors use `sudo chmod` to fix?!.

Reply:
done